src/components/dataTable/DataTable.js

```diff
@@ -96,7 +96,7 @@ const DataTable = ({
   }
 
   // the initial fetch is happening, show spinner, don't render the table yet
-  if (isLoading || isFetching) {
```

Comment:
I believe you want the isFetching because when they hit the second page this needs to turn on.